integreat_cms/core/signals/auth_signals.py

```diff
@@ -48,7 +48,7 @@ def user_logged_in_callback(
 # pylint: disable=unused-argument
 @receiver(user_logged_out)
 def user_logged_out_callback(
-    sender: ModelBase, request: HttpRequest | HttpRequest, user: User, **kwargs: Any
+    sender: ModelBase, request: HttpRequest, user: User, **kwargs: Any
```

Comment:
Was this second one meant to have been [`django.http.HttpResponse`](https://docs.djangoproject.com/en/stable/ref/request-response/#httpresponse-objects)?

Reply:
No, I think this was a remnant of some more fine-graded typing like `WSGIRequest | ASGIRequest` which I overlooked during a global search/replace...
So your suggestion looks good!